server/routes/index.js

```diff
@@ -4,13 +4,14 @@ var loginRouter = require('./login/login.route');
 var taskBuilderData = require('./taskBuilderData');
 var skillRoutes = require('./skill.routes');
 var verifyToken = require('./login/verifyToken');
-
+var userData = require('./userData');
 module.exports = function () {
 	router.use('/login', loginRouter());
 
 	router.use('*', verifyToken);
 	router.use('/skill', skillRoutes);
   router.use('/fetchTaskData', taskBuilderData);	
+	router.use('/user', userData);	
```

Comment:
If all users should be allowed to access this API or only admin users should be allowed
Baban sir to confirm.